angular/projects/admin-nrpti/src/app/records/records-resolver.ts

```diff
@@ -55,6 +56,15 @@ export class RecordsResolver implements Resolve<Observable<object>> {
 
     if (params.act) {
       or['legislation.act'] = params.act;
+      // adds the actCode associated with each act to the query obj
```

Comment:
We aren't repeating this code like above but could also be a util. Again easier to test and update.

Reply:
created a function in record-utils to do this and refactored.